src/main/daml/ContingentClaims/Core/Internal/Claim.daml

```diff
@@ -137,7 +137,7 @@ instance (Show t, Show x, Show a, Show o) => Show (Claim t x a o) where
     AnytimeF b e -> print2 "Anytime" b $ printElectable e
     UntilF b s -> print2 "Until" b s
     where
-      printElectable (t,s) = t <> ": " <> s
+      printElectable (t, s) = t <> ": " <> s
```

Comment:
Is this part of the styleguide?

Reply:
Just to be consistent with the rest of the tuples in the file.